superset/embedded_dashboard/__init__.py

```diff
@@ -1,16 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Any reasons for deleting this? Can you bring it back?

Reply:
I think this might be used by some organizations so we don't want to delete it if it is not necessary

Reply:
If this folder is used, it would be good to add some comment or README file. Empty folders can be deleted at any time during cleanup efforts.